docs/source/methods/method_of_characteristics.rst

```diff
@@ -261,7 +261,7 @@ The subscripts :math:`m` and :math:`p` for the azimuthal and polar angles, respe
 The Flat Source Region Approximation
 ====================================
 
-Another common approximation for MOC is to assume that the source :math:`Q_g` is constant across discrete spatial cells termed *flat source regions* (FSRs). This implies that the source does not vary along a characteristic :math:`k` entering FSR :math:`i` at :math:`s'` and exiting at :math:`s''`: 
+Another common approximation for MOC is to assume that the source :math:`Q_g` is constant across discrete spatial cells termed *source regions* (SRs). This implies that the source does not vary along a characteristic :math:`k` entering FSR :math:`i` at :math:`s'` and exiting at :math:`s''`:
```

Comment:
In this case I think that "flat source region" is appropriate based on the title of the section